plugins/rts54hub/fu-rts54hub-rtd21xx-mergeinfo.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 Ricky Wu <ricky_wu@realtek.com> <spring1527@gmail.com>
+ * Copyright 2021 Richard Hughes <richard@hughsie.com>
```

Comment:
Is this copyright string correct? I'd expect it to be 2025 Realtek or something.